### PR TITLE
Fix OVA's internet issue

### PR DESCRIPTION
**Pull request by pranavmalaviya**
This PR fixes the issue of OVA not having access to the internet. We had a flag [here](https://github.com/chef/bento/blob/main/packer_templates/ubuntu/ubuntu-18.04-amd64.json#L99) to remove ethernet interfaces. This was added because we were looking forward to creating Vagrant boxes from this ISO when developing locally. More details on this are shared internally in a document.

### CODE REVIEW

File: vmware/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+3. (Optional) If you want to follow the build process in GUI, then you need to turn the `headless` bool to `false` in the `ubuntu-18.04-amd64.json` file.
+
+4. Build the packer template by running this command
+
+   ```sh
+   packer build ubuntu-18.04-amd64.json
+   ```
+
+5. This will take 20 to 80 minutes based on your OS and machine. Grab a coffee and appreciate life. At the end of the process, the OVA will be in the `./builds/packer-ubuntu-18.04-amd64-vmware` directory along with a few other files. If you're running the packer template again, the `packer-ubuntu-18.04-amd64-vmware` directory must not exist or be empty.
```
Comment (jmariomejiap) [inline vmware/README.md:40]: ☕

**Review by jmariomejiap — APPROVED** [review]
Nice changes @pranavmalaviya. I might have a few questions about the scripts we are including. I read the documentation and ask you questions if needed.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Infrastructure-templates
 
-A collection of Packer templates to create AWS AMIs with our projects installed and configured. This repository contains packer templates created using [amazon-ebs](https://www.packer.io/docs/builders/amazon/ebs) builder. It leverages the AWS EC2 and creates an Amazon Machine Image (AMI).
+A collection of Packer templates to create AWS AMIs and VMware image with our projects installed and configured. This repository contains packer templates created using [amazon-ebc](https://www.packer.io/docs/builders/amazon/ebs) and [vmware-iso](https://www.packer.io/docs/builders/vmware/iso) builders.
```
[the remark this answers is not in the document]
Comment (pranavmalaviya) [reply README.md:3]: Oh. Thanks for catching this. :)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
 
-Requirements and steps to run this packer template is inside `aws` folder.
+The [amazon-ebc](https://www.packer.io/docs/builders/amazon/ebs) builder leverages the AWS EC2 and creates an Amazon Machine Image (AMI), while the [vmware-iso](https://www.packer.io/docs/builders/vmware/iso) builder uses VMware tools and exports an OVA file.
```
[the remark this answers is not in the document]
Comment (pranavmalaviya) [reply README.md:5]: So, I did a typo and AWS already had that service for that acronym 😂

**Comment by pranavmalaviya** [comment]
@SamDesmondKing Most of the changes here are because of reverting #18, which I agree can be done on its own PR. I'll make a separate PR before merging this one in.

**Pull request merged**